### PR TITLE
Add Multiple tables queries pull request

### PR DESCRIPTION
**Pull request by AleWaweru**
- Add Multiple tables queries using primary key and foreign keys

Screenshot (1)
Screenshot (2)
Screenshot (3)
Screenshot (4)
Screenshot (5)
Screenshot (6)
Screenshot (7)
Screenshot (8)
Screenshot (9)
Screenshot (10)
Screenshot (11)
Screenshot (12)
Screenshot (13)
Screenshot (14)

### CODE REVIEW

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @AleWaweru , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

- [ ] Please make sure that everything that creates or change the tables structure should be saved in the `schema.sql` file. Currently you have no SQL command in your `schema.sql` that create the `owners` table, create the `species` table and modify the animal table as require in the project requirement. :point_down: 
  - Create a table named `owners` with the following columns:
    - `id`: integer (set it as autoincremented PRIMARY KEY)
    - `full_name`: string
    - `age`: integer
  - Create a table named `species` with the following columns:
    - `id`: integer (set it as autoincremented PRIMARY KEY)
    - `name`: string
  - Modify `animals` table:
    - Make sure that id is set as autoincremented PRIMARY KEY
    - Remove column `species`
    - Add column `species_id` which is a foreign key referencing `species` table
    - Add column `owner_id` which is a foreign key referencing the `owners` table
  - **Remember all these goes in `schema.sql` file**.

- [ ] Please make sure that everything that populates tables with data should be saved in the `data.sql` as require in the project requirement, you should :point_down: 
  
  - Insert the following data into the `owners` table:
    - Sam Smith 34 years old.
    - Jennifer Orwell 19 years old.
    - Bob 45 years old.
    - Melody Pond 77 years old.
    - Dean Winchester 14 years old.
    - Jodie Whittaker 38 years old.
  - Insert the following data into the `species` table:
    - Pokemon
    - Digimon
  - Modify your inserted animals so it includes the `species_id` value:
    - If the name ends in "mon" it will be Digimon
    - All other animals are Pokemon
  - Modify your inserted animals to include owner information (`owner_id`):
    - Sam Smith owns Agumon.
    - Jennifer Orwell owns Gabumon and Pikachu.
    - Bob owns Devimon and Plantmon.
    - Melody Pond owns Charmander, Squirtle, and Blossom.
    - Dean Winchester owns Angemon and Boarmon.
  - **Remember these insertions and modifications happen in `data.sql` file.**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahmetbozaci** [comment]
### Hello @AleWaweru 
### _This is an additional review as a result of the second opinion request._ 


- [ ] There are no changes in the schema.sql file. Kindly implement [these](https://github.com/AleWaweru/vet-clinic-database/pull/4#:~:text=Please%20make%20sure%20that%20everything,in%20schema.sql%20file.) [Requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=Create%20a%20table,schema.sql%20file.)
image 
- It looks like you implement [these](https://github.com/AleWaweru/vet-clinic-database/pull/4#:~:text=Please%20make%20sure%20that%20everything%20that%20populates,and%20modifications%20happen%20in%20data.sql%20file.) changes in the data.sql file after the review.

___


### Cheers and Happy coding! 💻 👏
____
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: schema.sql [hunk 1]
```diff
@@ -9,3 +9,27 @@ CREATE TABLE animals (
...
+
+ALTER TABLE animals
+ADD COLUMN species_id INTEGER REFERENCES species (id);
+
+ALTER TABLE animals
+ADD COLUMN owner_id INTEGER REFERENCES owners (id);
+
+
+
+SELECT * FROM animals;
```
Comment (TadesseDev) [inline schema.sql:38]: - [Optional] Dear @AleWaweru, Please remember everything that is a query that helps to analyze data should be saved in the queries.sql file. Hence the `select` statement should be written in the `queries.sql` file.

File: schema.sql [hunk 2]
```diff
@@ -9,3 +9,27 @@ CREATE TABLE animals (
...
+CREATE TABLE species (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255)
+);
+
+
+ALTER TABLE animals
+DROP COLUMN species;
+
+ALTER TABLE animals
```
Comment (TadesseDev) [inline schema.sql:30]: - [ ] Good job adding the required columns and doing most of the highlighted changes by @ahmetbozaci, But I notice you have missed one of the requirements, which is to set the id as autoincremented PRIMARY KEY for the animal table. Please remember we are required to `Make sure that id is set as autoincremented PRIMARY KEY`.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @AleWaweru 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

199733072-bdacca19-8df9-43a7-be93-3aff2f1c15c8

 ## Highlights
 - [x] Good job making most of the changes requested by the previous reviewer. 🙏


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (AleWaweru) [reply schema.sql:38]: Done.

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- You meet all the requested changes by the previous reviewer. ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**